feat/src/main/java/com/example/feat/document/DocumentScreen.kt

```diff
@@ -0,0 +1,185 @@
+            ) {
+                Row {
+                    Image(
+                        painterResource(R.drawable.ic_logo),
+                        contentDescription = "",
+                        contentScale = ContentScale.Crop,
+                        modifier = Modifier.size(30.dp),
+                    )
+                    Spacer(modifier = Modifier.width(5.dp))
+                    Text(
+                        text = " Tutor",
+                        style = DesignSystem.TITLE_LARGE_STYLE.copy(
+                            color = colorResource(R.color.primaryColor),
+                            fontSize = 24.sp,
+                        ),
+                    )
+                }
+                Row {
+                    Icon(
+                        Icons.Default.Search,
+                        contentDescription = "Search",
+                        modifier = Modifier.size(24.dp),
+                        tint = Color.Black.copy(alpha = 0.7f)
+                    )
+                    Spacer(modifier = Modifier.width(20.dp))
+                    Icon(
+                        Icons.Default.FavoriteBorder,
+                        contentDescription = "Search",
+                        modifier = Modifier.size(24.dp),
+                        tint = Color.Black.copy(alpha = 0.7f)
+                    )
+                }
+            }
+        }
+    ) { innerPadding ->
+        Column(
+            modifier = Modifier
+                .padding(innerPadding)
+                .fillMaxSize()
+        ) {
+            if (uiState.isLoading) {
+                Skeleton(
+                    width = screenWidth.toDouble(), height = 250.0, leftPadding = 10.0,
+                    rightPadding = 10.0,
+                )
+                VerticalList(size = 7) {
+                    Skeleton(
+                        width = screenWidth.toDouble(),
+                        height = 150.0,
+                        leftPadding = 10.0,
+                        rightPadding = 10.0,
+                        topPadding = 10.0,
+                        bottomPadding = 10.0,
+                    )
+                }
+            } else {
+                Box(
+                    modifier = Modifier
+                        .padding(horizontal = 10.dp) // padding outside box
+                        .clip(RoundedCornerShape(5.dp))
+                        .background(colorResource(id = R.color.primaryColor))
+                        .fillMaxWidth()
+                        .padding(15.dp),
+                    contentAlignment = Alignment.Center
+                ) {
+                    Column(
+                        modifier = Modifier.fillMaxWidth(),
+                        horizontalAlignment = Alignment.CenterHorizontally
+                    ) {
+                        Timer(
+                            millisInFuture = uiState.bookingInfo?.scheduleDetailEntity?.startPeriodTimestamp ?: 0L,
```

Comment:
Using a default value of 0L for `millisInFuture` might lead to incorrect timer behavior if the timestamp is not available. Consider handling this case differently.

```diff
-                            millisInFuture = uiState.bookingInfo?.scheduleDetailEntity?.startPeriodTimestamp ?: 0L,
+                            millisInFuture = uiState.bookingInfo?.scheduleDetailEntity?.startPeriodTimestamp ?: return,
```

Reply:
I set the default is OL to avoid crashing, in component Timer i have set the condition:
```
if (currentTime > millisInFuture) {
        return
}
```
so I think it will be correct